packages/react-native-editor/package.json

```diff
@@ -1,123 +1,122 @@
 {
-  "name": "@wordpress/react-native-editor",
```

Comment:
I saved this file with IDE and `.editorconfig` kicked in. The best way to review is to use:
![Screen Shot 2020-06-29 at 09 04 57](https://user-images.githubusercontent.com/699132/85983333-a41a6a80-b9e7-11ea-951d-56a8a2484e81.png)